examples/jsm/renderers/webgpu/WebGPUBackend.js

```diff
@@ -342,15 +347,15 @@ class WebGPUBackend extends Backend {
 
 		if ( renderContext.viewport ) {
 
-			this.updateViewport( renderContext );
+			this.updateViewport( renderContext, bufferHeight );
```

Comment:
I think we can add the properties `height` and `width` in `RenderContext` and remove `bufferHeight`?

Reply:
I'll take a look.